src/org/benf/cfr/reader/bytecode/analysis/variables/Keywords.java

```diff
@@ -19,8 +19,30 @@
             "const", "float", "native", "super", "while",
             "true", "false", "null" // Not keywords (see url), but literals you can't use.
     );
+    
+    /**
+     * Replacements for the keywords list, in order
+     * Tries to use common given variable names
+     */
+    private static final List<String> replacements =  ListFactory.newList(
+        "abstrct", "_continue", "_for", "_new", "zwitch",
+        "asert", "deflt", "_goto", "pckg", "synched",
+        "bool", "_do", "_if", "privt", "that",
+        "_break", "doubleVal", "impl", "protctd", "_throw",
+        "byteVal", "elze", "imprt", "pblc", "throwz",
+        "caze", "enumVal", "instanzeof", "ret", "tranzient",
+        "_catch", "extendz", "intVal", "shortVal", "_try",
+        "_char", "_final", "interfaze", "_static", "_void",
+        "clazz", "finaly", "longVal", "strict", "volatil",
+        "constant", "floatVal", "_native", "_super", "_while",
+        "tru", "fals", "nullVal" 
+    );
```

Comment:
I get where you're going with this, but it's a really bad idea to assume that people pay enough attention to detail to keep two unrelated lists in synch.  
Sorry, zynch.

;)

Better to use a map which is statically constructed - I'll accept this change then tweak it to do that.

Reply:
Though I'm not sure I'm with you on half of these.   Will ponder..............

Reply:
As i said, it's only a draft of how it could look like ;) Change what you want